History.md

```diff
@@ -1,3 +1,27 @@
+### 1.12 / 2016-05-27
+
+*   4 major enhancements:
+
+    *   Replace the use of ObjectSpace with a custom system to get a list of the
+        children of StandardError.  This makes the library JRuby-compatible.
```

Comment:
Two small errors. First, `. `, not `.  `. See [Farhad Manjoo](http://www.slate.com/articles/technology/technology/2011/01/space_invaders.html) on this matter.

Second, the second sentence is better stated as: This makes the provided rake tasks compatible with JRuby.